Update CLI Commands with upgrade commands (#1264)
ava-labs/avalanche-docs: docs/subnets/reference-cli-commands.md

<!-- markdownlint-enable MD013 -->

#### Import from a Public Network

The `subnet import public` command imports a Subnet configuration from a running network.

The genesis file should be available from the disk for this to work. By default, an imported Subnet

> doesn't overwrite an existing Subnet with the same name. To allow overwrites, provide the --force

Reviewer: ```suggestion
doesn't overwrite an existing Subnet with the same name. To allow overwrites, provide the `--force`
```

### Subnet VMID

The `subnet vmid` command prints the virtual machine ID (VMID) for the given Subnet.

**Usage:**

```shell
avalanche subnet vmid [subnetName]
```

## Subnet Upgrade

The `subnet upgrade` command suite provides a collection of tools for
updating your developmental and deployed Subnets.

### Subnet Upgrade Apply

Apply generated upgrade bytes to running Subnet nodes to trigger a network upgrade.

For public networks (Fuji Testnet or Mainnet), to complete this process, you must have access to the
machine running your validator. If the CLI is running on the same machine as your validator, it can
manipulate your node's configuration automatically. Alternatively, the command can print the
necessary instructions to upgrade your node manually.

After you update your validator's configuration, you need to restart your validator manually. If you
provide the `--avalanchego-chain-config-dir` flag, this command attempts to write the upgrade file

> at that path. Refer to [this doc](../nodes/maintain/chain-config-flags#subnet-chain-configs) for

Reviewer: ```suggestion
at that path. Refer to [this doc](../nodes/maintain/chain-config-flags.md#subnet-chain-configs) for
```

### Subnet VMID

The `subnet vmid` command prints the virtual machine ID (VMID) for the given Subnet.

**Usage:**

```shell
avalanche subnet vmid [subnetName]
```

## Subnet Upgrade

The `subnet upgrade` command suite provides a collection of tools for
updating your developmental and deployed Subnets.

### Subnet Upgrade Apply

Apply generated upgrade bytes to running Subnet nodes to trigger a network upgrade.

For public networks (Fuji Testnet or Mainnet), to complete this process, you must have access to the
machine running your validator. If the CLI is running on the same machine as your validator, it can
manipulate your node's configuration automatically. Alternatively, the command can print the
necessary instructions to upgrade your node manually.

After you update your validator's configuration, you need to restart your validator manually. If you
provide the `--avalanchego-chain-config-dir` flag, this command attempts to write the upgrade file at
that path. Refer to

> <https://docs.avax.network/nodes/maintain/chain-config-flags#subnet-chain-configs> for related

Reviewer: ```suggestion
[this doc](../nodes/maintain/chain-config-flags.md#subnet-chain-configs) for related
```
Make sure the link is correct.

    --local local                           apply upgrade existing local deployment
    --mainnet mainnet                       apply upgrade existing mainnet deployment
    --print                                 if true, print the manual config without prompting (for public networks only)
    --testnet testnet                       apply upgrade existing testnet deployment (alias for `fuji`)
```

### Subnet Upgrade Export

Export the upgrade bytes file to a location of choice on disk.

**Usage:**

```shell
avalanche subnet upgrade export [subnetName] [flags]
```

**Flags:**

```shell
    --force                     If true, overwrite a possibly existing file without prompting
-h, --help                      help for export
    --upgrade-filepath string   Export upgrade bytes file to location of choice on disk
```

### Subnet Upgrade Generate

The `subnet upgrade generate` command builds a new upgrade.json file to customize your Subnet. It
guides the user through the process using an interactive wizard.

**Usage:**

```shell
avalanche subnet upgrade generate [subnetName] [flags]
```

**Flags:**

```shell
-h, --help   help for generate
```

### Subnet Upgrade Import

> Import the upgrade bytes file into the local environment

Reviewer: ```suggestion
Import the upgrade bytes file into the local environment.
```

**Usage:**

```shell
avalanche subnet upgrade import [subnetName] [flags]
```

**Flags:**

```shell
-h, --help                      help for import
    --upgrade-filepath string   Import upgrade bytes file into local environment
```

### Subnet Upgrade Print

> Print the upgrade.json file content

Reviewer: ```suggestion
Print the upgrade.json file content.
```
